contracts/contracts/coordination/BqETHSubscription.sol

```diff
@@ -124,23 +181,64 @@ contract BqETHSubscription is IFeeModel {
         activeRitualId = INACTIVE_RITUAL_ID;
     }
 
-    function packageFees() public view returns (uint256) {
-        return feeRate * maxDuration * maxNodes;
+    function baseFees() public view returns (uint256) {
+        return baseFeeRate * packageDuration * maxNodes;
+    }
+
+    function encryptorFees(uint128 encryptorSlots, uint32 duration) public view returns (uint256) {
+        return encryptorFeeRate * duration * encryptorSlots;
     }
 
     /**
      * @notice Pays for a subscription
```

Comment:
Something like this, but with more flow 😂  we need a native speaker here to chime in:

```suggestion
     * @notice Pays for the closest unpaid subscription period (either the current or the next)
```